src/app/auth/login/login.component.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { UsuarioService } from 'src/app/services/usuario.service';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { RouterTestingModule } from "@angular/router/testing";
+
+import { LoginComponent } from './login.component';
+import { Observable } from 'rxjs';
+
+describe('LoginComponent', () => {
+  let usuarioService:UsuarioService
+  let fb:FormBuilder
+  let component: LoginComponent;
+  let fixture: ComponentFixture<LoginComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ LoginComponent ],
+      
+      imports: [
+        RouterTestingModule,
+        HttpClientTestingModule,
+        ReactiveFormsModule
+      ]
+    })
+    .compileComponents();
+    fixture = TestBed.createComponent(LoginComponent);
+    component = fixture.componentInstance;
+    usuarioService = TestBed.inject(UsuarioService)
+    fb = TestBed.inject(FormBuilder)
+    fixture.detectChanges();
+  });
+
+  describe('Formulario', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
  describe("Formulario", () => {
```